README.md

```diff
@@ -38,47 +50,22 @@ Once you have your dependencies installed, you can run:
 npm run build
 ```
 
-The output library will be in the `/dist` directory. `gz3d.js` is good for developing purposes, as it's not minified. The minified version, `gz3d.min.js` should be used in production environments.
-
-## Local testing
+## Local development
 
-If you want to try gzweb in an application without publishing, then you can
-setup a `link` using these steps.
+If you are working on changes on `gzweb` and want to try them in an application, then you can setup a `link` using these steps.
 
 1. In the root of the `gzweb` sources run
-  ```
-  npm link
-  ```
+    ```
+    npm link
+    ```
 
-2. This should create a symlink to the `gzweb` folder in the global node
-   path, which can be found using `npm root -g`.
+2. This should create a symlink to the `gzweb` folder in the global node path, which can be found using `npm root -g`.
 
 3. Connect the downstream application to the `gzweb` symlink using
     ```
     npm link gzweb
     ```
 
-4. In the downstream application's `node_modules` directory you should see
-   a corresponding symlink to `gzweb`.
+4. In the downstream application's `node_modules` directory you should see a corresponding symlink to `gzweb`.
 
 5. All you have to do now is run `npm run build`.
-
-## Publish
```

Comment:
Can this be kept, maybe under a "## Maintainer" section?

Reply:
As this is published in NPM, probably we should keep the information on how to make releases on a `PUBLISHING.md` file alongside the `README.md`. I've seen it before on other repos (some examples [here](https://github.com/markedjs/marked/blob/master/docs/PUBLISHING.md) and [here](https://github.com/opticdev/optic-node/blob/main/PUBLISHING.md)).

Still, I'm more than happy to keep things simple and add this info in the `README`. We can always create other files in the future as we see fit.